doc/source/f2py/buildtools/cmake.rst

```diff
@@ -0,0 +1,60 @@
+.. _f2py-cmake:
+
+===================
+Using via ``cmake``
+===================
+
+In terms of complexity, ``cmake`` falls between ``make`` and ``meson``. The
+learning curve is steeper since CMake syntax is not pythonic and is closer to
+``make`` with environment variables.
+
+However, the trade-off is enhanced flexibility and support for most architectures
+and compilers. An introduction to the syntax is out of scope for this document,
+but the `official CMake Tutorials`_ are a fantastic resource.
```

Comment:
I don't get this line. It seems like there's wide consensus that the the official tutorials and examples of CMake suck, badly. And browsing it again now, I tend to agree. Why not link to, e.g., https://cliutils.gitlab.io/modern-cmake/?

Reply:
^_^ Somehow I got by on the official tutorial and references; these are much better though, updating.